ocw/lib/azure.py

```diff
@@ -85,13 +98,19 @@ def cleanup_all(self):
         storage_keys = [v.value for v in storage_keys.keys]
         block_blob_service = BlockBlobService(account_name='openqa', account_key=storage_keys[0])
 
+        keep_images = self.get_keeping_images()
+        self.cleanup_sle_images_container(block_blob_service, keep_images)
+        self.cleanup_disks_from_rg(keep_images)
+        self.cleanup_images_from_rg(keep_images)
+
+        for i in keep_images:
+            logger.info("[Azure] Keep image {} ".format(i))
+
         containers = block_blob_service.list_containers()
```

Comment:
in case this collection needed only for `cleanup_bootdiagnostics_container` maybe we should get it inside this function and looping also inside function ? this would make code flow more clear

Reply:
ack